chardet/big5freq.py

```diff
@@ -381,545 +381,6 @@
  938,3941, 553,2680, 116,5783,3942,3667,5784,3545,2681,2783,3438,3344,2820,5785, # 5328
 3668,2943,4160,1747,2944,2983,5786,5787, 207,5788,4809,5789,4810,2521,5790,3033, # 5344
  890,3669,3943,5791,1878,3798,3439,5792,2186,2358,3440,1652,5793,5794,5795, 941, # 5360
-2299, 208,3546,4161,2020, 330,4438,3944,2906,2499,3799,4439,4811,5796,5797,5798, # 5376  #last 512
```

Comment:
It was all commented out in the upstream code, so it just seemed like a waste of space/memory to keep them.  The comment seems to indicate that nothing after that point occurs is one of the 512 most frequent characters for Big5.